operator/tests/central/basic/74-assert.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: stackrox-db
+spec:
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 100Gi
```

Comment:
nit: some files has EOL and some not

Reply:
Thanks, CI's style check fortunately also complains about such things. Fixed it.